Marlin/src/HAL/HAL_AVR/MarlinSerial.cpp

```diff
@@ -155,7 +157,7 @@
                 wait_for_user = wait_for_heatup = false;
                 break;
               case state_M112:
-                kill(PSTR(MSG_KILLED));
+                ++killed_by_M112;
```

Comment:
Incrementing bools to set them true is deprecated in C++11, and odd?

Reply:
Deprecated? Well in that case…